config/puma.rb

```diff
@@ -42,3 +42,11 @@
 
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
+
+pidfile "/tmp/puma.pid"
```

Comment:
If we save it in the rails tmp dir an unclean exit of the container on the development environment will fail to start again, and regarding production it doesn't affect, I'll leave it at `/tmp`...